tests/e2e/cmp_test.go

```diff
@@ -364,7 +364,11 @@ var _ = Describe("odoCmpE2e", func() {
 	Context("cleaning up", func() {
 		It("should delete the application", func() {
 			runCmd("odo app delete " + appTestName + " -f")
-			runCmd("oc delete project " + projName)
+
+			runCmd("odo project delete " + projName)
+			time.Sleep(15 * time.Second)
```

Comment:
adding a sleep timer is bad / adds unnecessary wait. You may have to implement / use some form of ping for this to work. Besides, the project should already be deleted immediately instead of having to wait...

Reply:
Our only sleep timer within the e2e tests is here: https://github.com/redhat-developer/odo/blob/c5ffed5dc67b2fcf22a0a7d3a360d34d345bebba/tests/e2e/e2e.go#L27 which is part of ping.

Reply:
Ack Done